gen/go/javaagent/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/Traceableai/agent-config/gen/go/javaagent
```

Comment:
this was auto generated code, no idea how this got created just for this message

Reply:
went away after merging main